### ISSUE TITLE
The newest ver crashes the game on 1.21

### ISSUE THREAD

**Issue by BillCipher345** [Grayray75/FPS-Display#19]
What the title says, it just makes ur game crash, pls fix that, i gotta use the old ver now.

**Comment by Grayray75** [comment]
Hi there, can you please provide a bit more details like the Minecraft version you are using and the crash log.

**Comment by BillCipher345** [comment]
I play on 1.21, and when using the latest version (which is listed as 1.21.x) it crashes, here is the crash log:
mclo.gs/P0yYxEB

**Comment by Grayray75** [comment]
This is strange, this is indeed caused by my mod.
The crash only happens when using newer fabric-loader version like 0.16.9, but not on 0.15.11, that's why I didn't catch that while testing...

For now just use v4.2.1 until I find a fix for this

**Comment by BillCipher345** [comment]
Yea thats what i do, it is just annoying that i see the mod menu update checker says there is a mod to update while it is this mod lol, and also every time i now do "check for updates" on prism launcher i gotta uncheck ur mod so it wont update it.
Anyway ty

**Comment by BillCipher345** [comment]
Oh now i see u made the latest ver to be 1.21.2-.3 instead of 1.21.x, thanks! That makes it so i wont see the "update available" on mod menu!

**Comment by Grayray75** [comment]
Yeah it seems like the latest version is not compatible with 1.21.0 ... if other mods are installed.
And this issue might have discovered a bug in the fabric-loader lol.

**Comment by Grayray75** [comment]
I don't remember the exact details of this issue (*it's been more than a year lol*).
But I think this was something that was only partially caused by my mod and not really fixable.
